multiraft/multiraft.go

```diff
@@ -357,6 +359,17 @@ func (m *MultiRaft) ChangeGroupMembership(groupID proto.RangeID, commandID strin
 	return ch
 }
 
+// CoincideRaftLeader try to make the raft leader located to the same
```

Comment:
I'm not sure the comments here in Multiraft package should mention the "range leader" concept, especially since this function does not consider the range leader status.

What this method does is, for the given groupID, move this node to the Candidate state iff it is not currently the leader of the group. It makes no guarantee that the current node will win the ensuing election, although it might.

So a better method name in this package would be "CampaignIfNotLeader", or maybe simply "Campaign".